CHANGELOG.md

```diff
@@ -6,32 +6,24 @@
-  - Enable std feature for indexmap (#1709). [[@jstarry], [#1709](https://github.com/yewstack/yew/pull/1709)]
-  - Added missing KeyboardService re-export (#1694). [[@SOF3], [#1694](https://github.com/yewstack/yew/pull/1694)]
-  - Clean up component lifecycle state (#1700). [[@jstarry], [#1700](https://github.com/yewstack/yew/pull/1700)]
-  - Move top-level crates to packages/<crate> (#1680). [[@philip-peterson], [#1680](https://github.com/yewstack/yew/pull/1680)]
-  - Refactor component lifecycle event handling (#1692). [[@jstarry], [#1692](https://github.com/yewstack/yew/pull/1692)]
-  - Prune stdweb examples to reduce maintenance burden (#1690). [[@jstarry], [#1690](https://github.com/yewstack/yew/pull/1690)]
-  - Refactor html module into new component submodule (#1689). [[@jstarry], [#1689](https://github.com/yewstack/yew/pull/1689)]
-  - Rename internal Agent structs to match Component (#1688). [[@jstarry], [#1688](https://github.com/yewstack/yew/pull/1688)]
-  - Revert "Update rand requirement from 0.7 to 0.8 (#1682)" (#1684). [[@siku2], [#1682](https://github.com/yewstack/yew/pull/1682)]
-  - Add discussion link to issue selector (#1674). [[@jstarry], [#1674](https://github.com/yewstack/yew/pull/1674)]
-  - Update link to Material Design Components (#1662). [[@TapioT], [#1662](https://github.com/yewstack/yew/pull/1662)]
-  - Extract Classes to a separate macro (#1601). [[@cecton], [#1601](https://github.com/yewstack/yew/pull/1601)]
-  - Improve the "keyed_list" example (#1650). [[@titaneric], [#1650](https://github.com/yewstack/yew/pull/1650)]
-  - Apply new Clippy lints, examples deployment, and stdweb integration tests (#1651). [[@siku2], [#1651](https://github.com/yewstack/yew/pull/1651)]
-  - Add documentation for component children (#1616). [[@K4rakara], [#1616](https://github.com/yewstack/yew/pull/1616)]
-  - More ergonomic use state 1505 (#1630). [[@mattferrin], [#1630](https://github.com/yewstack/yew/pull/1630)]
-  - Remove Drop bound from Task trait (#1627). [[@siku2], [#1627](https://github.com/yewstack/yew/pull/1627)]
-  - Document dynamic tag names (#1628). [[@siku2], [#1628](https://github.com/yewstack/yew/pull/1628)]
-  - Add a macro for building properties outside of html! (#1599). [[@siku2], [#1599](https://github.com/yewstack/yew/pull/1599)]
+  - Implicit optional attributes. [[@siku2](https://github.com/siku2), [#1637](https://github.com/yewstack/yew/pull/1637)]
+  - Added callback_future_once in yewtil.(#1712). [[@fraillt](https://github.com/fraillt), [#1696](https://github.com/yewstack/yew/pull/1696)]
+  - Added relevant examples section to the docs. [[@oOBoomberOo](https://github.com/oOBoomberOo), [#1695](https://github.com/yewstack/yew/pull/1695)]
+  - Added missing KeyboardService re-export. [[@SOF3](https://github.com/SOF3), [#1694](https://github.com/yewstack/yew/pull/1694)]
+  - Clean up component lifecycle state. [[@jstarry](https://github.com/jstarry), [#1700](https://github.com/yewstack/yew/pull/1700)]
```

Comment:
Is this really a feature? or a fix?

Reply:
Wait... I meant to only select L19... Don't know why it selected many of them

Reply:
No worry I know immediately it's line 19

:thinking: I'm not even sure if it's a fix or if it's relevant for the user.

Reply:
```suggestion
```